DuckDuckGo/NetworkProtectionStatusView.swift

```diff
@@ -74,6 +98,49 @@ struct NetworkProtectionStatusView: View {
     }
 }
 
+private extension View {
+    @ViewBuilder
+    func hideScrollContentBackground() -> some View {
```

Comment:
This is to override the system list background colour. Quite an ugly solution pre-iOS 16 but not sure how else to achieve it...